src/core.js

```diff
@@ -178,7 +178,9 @@ QUnit.equiv = equiv;
 QUnit.dump = dump;
 
 // 3.0 TODO: Remove
-QUnit.jsDump = dump;
+Object.defineProperty( QUnit, "jsDump", {
+    get: function() { throw new Error( "Use QUnit.dump instead of QUnit.jsDump" ); }
```

Comment:
I hope I did but there can be mistakes I know please feel free to point out . . .